lib/multiboats.rb

```diff
@@ -0,0 +1,39 @@
+class Multiboats
+
+  attr_reader :ships
+  def initialize(loop_num)
+    @loop_num = loop_num
+    @ships = []
+  end
+
+  def ship_name_input
+    p "Enter a ship name (string)"
+    @ship_name = gets.chomp
+  end
+
+  def ship_length_input
+    p "Enter #{@ship_name} length (4 or less)"
+    @ship_length = gets.chomp.to_i
+    until @ship_length.between?(1,4)
+      p "Invalid"
+      p "Enter #{@ship_name} length (4 or less)"
+      @ship_length = gets.chomp.to_i
+    end
+  end
+
+  def create_ship
+    ship1 = Ship.new(@ship_name, @ship_length)
+    @ships << ship1
+  end
+
+  def ship_loop
```

Comment:
I really like this idea, of how you used the times method to allow for multiple ships